content/docs/intro/concepts/state.md

```diff
@@ -139,221 +98,167 @@ Enter your access token from https://app.pulumi.com/account/tokens
 
-If you are currently using a self-managed backend, but would now prefer to
-simplify things, just run `pulumi logout` and then `pulumi login`, and you’ll be back to
-using `app.pulumi.com`.
+<img src="/images/docs/reference/state_saas.png" alt="Pulumi Service Architecture" class="img-bordered">
 
-If you'd like to migrate your stacks from a self-managed backend to the Pulumi Service
-backend, you can follow the steps below. Suppose the stack "my-app-production"
-has been managed with a self-managed backend, and you want to migrate it to
-the Pulumi Service. Run the following commands if you are already logged in to the local
-endpoint:
+The Pulumi Service doesn't ever acquire your cloud credentials, and does not communicate with your cloud provider directly. Instead, the CLI itself coordinates with both the Pulumi Service's API and your cloud provider's directly. This ensures your IAM and key management does not need to change while adopting Pulumi. In particular, if you are running Pulumi deployments from [within a CI/CD environment]({{< relref "/docs/guides/continuous-delivery" >}}), you can rely on existing mechanisms and security practices that your organization has already put in place. The Pulumi Service is reliable, secure, and has undergone multiple audits, including SOC2 and professional pen-testing.
 
-```sh
-# switch to the stack we want to export
-$ pulumi stack select my-app-production
+It is possible to host your own version of the Pulumi Service in your private cloud environment. Pulumi offers versions that run natively on AWS, Azure, GCP, Kubernetes, or simple virtual machine-based private and hybrid cloud environments. The architecture is very similar, but is privately hosted and does not depend on public access over the Internet:
 
-# export the stack's checkpoint to a local file
-$ pulumi stack export --show-secrets --file my-app-production.checkpoint.json
-$ pulumi logout
-$ pulumi login
+<img src="/images/docs/reference/state_enterprise.png" alt="Pulumi Enterprise Architecture" class="img-bordered">
 
-# create a new stack with the same name on pulumi.com
-$ pulumi stack init my-app-production
+To learn more about self-host options, see [Self-Hosted Pulumi Service]({{< relref "/docs/guides/self-hosted" >}}) or [Contact Us]({{< relref "/pricing#contact" >}}).
 
-# import the new existing checkpoint into pulumi.com
-$ pulumi stack import --file my-app-production.checkpoint.json
-```
+## Advanced State
+
+Pulumi is designed to abstract state management away from you so that you can operate in terms of declarative infrastructure as code. In certain advanced cases, you may want or need to interact with state more directly. In those cases, the following topics may be helpful.
```

Comment:
```suggestion
Pulumi is designed to abstract state management away from you so that you can operate in terms of declarative infrastructure as code. In certain advanced cases, you may want or need to interact with state more directly. In those cases, the following sections may be helpful.
```